less/uams.actionbar.less

```diff
@@ -80,7 +80,6 @@
 			    padding: 10px 0 0;
 			    margin: 10px 5% 0;
 			    font-size: 13px;
-			    line-height: 14px;
```

Comment:
I just removed this since it could inherit the line-height that wasn't inside a media query.